xml/obs_ag_spider_identification.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<section id="_obs_spider_identification">
+ <title>Spider Identification</title>
+ <para>
+   OBS identifys spiders by using the gem <command>voight_kampff</command>. Spiders will be identified to optimize the
+   page content for them.
+ </para>
+ <para>
+   To update the spider list, you must run the command <command>rake voight_kampf:import_user_agents</command>.
+   This downloads a list of spiders as a JSON file into the <filename>config/</filename> directory of the Rails application.
+ </para>
+ <para>
+   Switch to the <filename>config/</filename> directory and open the <filename>crawler-user-agents.json</filename> file with
```

Comment:
I agree to the first part but I don't think that it should matter us whether the user is sending a PR or not. This guide explains how to add a new spider and not how to contribute to a 3rd-party-project. I mean, not everyone who is using OBS knows what git is (neither GitHub and Pull Requests).